src/components/accordion/accordion.stories.ts

```diff
@@ -264,28 +264,42 @@ export const withIconStartAndEnd_TestOnly = (): string =>
     `
   );
 
-export const mediumIconForLargeAccordionItem_TestOnly = (): string => html`
-  <calcite-accordion scale="l" style="width: 400px">
-    <calcite-accordion-item heading="Accordion Item" scale="l"></calcite-accordion-item>
-    <calcite-accordion-item
-      heading="Accordion Item IconStart/End"
-      icon-start="plane"
-      icon-end="plane"
-      scale="l"
-    ></calcite-accordion-item>
-    <calcite-accordion-item heading="Accordion Item" scale="l" description="A great subtitle"></calcite-accordion-item>
-    <calcite-accordion-item
-      heading="Accordion Item IconStart/End"
-      icon-start="plane"
-      icon-end="plane"
-      scale="l"
-      description="A great subtitle"
-    ></calcite-accordion-item>
-    <calcite-accordion-item
-      heading="Accordion Item IconStart/End with a potentially two line title"
-      icon-start="banana"
-      icon-end="banana"
-      scale="l"
-    ></calcite-accordion-item>
-  </calcite-accordion>
+const useCases = [
```

Comment:
Incase someone wants to add more of these in the future maybe you could type `useCases`? Like...

`const useCases: {icon:string; heading:string; description: string;}`

Reply:
Kewl. I got two opinions in favor of plain HTML instead of this, what would you say in regard to that?

Reply:
IMO what you have is fine. Its why ES6 templates exist. Why duplicate the same HTML?